Sources/StreamChat/Models/BlockedUser.swift

```diff
@@ -0,0 +1,28 @@
+//
+// Copyright © 2024 Stream.io Inc. All rights reserved.
+//
+
+import Foundation
+
+/// A type representing a blocked user.
+public struct BlockedUser {
```

Comment:
Since we have `ChatUser` and `CurrentUser` with full user information available in these types then the `BlockedUser` is a bit different since it only includes id and blocked date. Should we rename it to `BlockedUserInfo` instead? @nuno-vieira

Reply:
Maybe `BlockedUserDetails` ? We also have `MessagePinDetails` and `MessageModerationDetails`. So maybe `Details` postfix has more consistancy, WDYT?